### ISSUE TITLE
Debugging with RobotOS

### ISSUE THREAD

**Issue by ga72kud** [jdlangs/RobotOS.jl#98]
I am using RobotOS.jl and like that package because it allows me to use JuliaLang for ROS. Usually, when I am coding in Julia and make an error, there will be an error message. When I am using RobotOS, this is not the case. Nothing happens, and it seems that it waits for a ROS message. 

Is there a possibility of changing the configurations, so I get error messages when something is not working? Getting an error message might be easier for me to get to a solution. Sometimes the program is not working, and I must search extensively for the problem. 

This problem is especially the case when I start the callbacks by RobotOS

**Comment by gdaddi** [comment]
I have the exact same problem. It is a nightmare to debug without any error feedback...

**Comment by ga72kud** [comment]
@gdaddi I agree and started a different program with RobotOSData.jl working only on rosbags. Maybe there is a workaround.
